Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
-import './Header.css';
+import './Header.sass';
+
+const HeaderContent = props => {
+    return (
+        <>
+            <p className="header-content">{props.content}</p>
+        </>
```

Comment:
Why use a fragment here? You're only returning one element, so just return the `p`.